assets/designer/environments/aml-designer/context/conda_dependencies.yaml

```diff
@@ -0,0 +1,31 @@
+name: aml-designer
+channels:
+- conda-forge
+dependencies:
+- python=3.8.10
```

Comment:
please upgrade python to at least 39

Reply:
The `azureml-designer-classic-modules` depend on the `pandas ==1.0.5` which is not support python 3.9 or higher .

I know the 3.8 will end of the support, maybe we can do this upgrade after the migration to system registry.